### ISSUE TITLE
Google maps iframe accessibility lacking name attr and hidden focus

### ISSUE THREAD

**Issue by Jkrzy** [18F/samhsa-prototype#186]
The `iframe` in which google maps is rendered on the /results page does not have a title or name attribute. Can this be resolved on our end? https://accessibility.18f.gov/iframes/

When navigating via keyboard, the iframe is also selectable although focus is hidden. 

_Originally posted by @Jkrzy in https://github.com/18F/samhsa-prototype/issues/184#issuecomment-518390655_

**Comment by Jkrzy** [comment]
Title added in #195.

Poked around a bit but not seeing a clear path to resolve the 2 tab stops, which are not visually represented, when selected via keyboard navigation associated with the map. 

Any thoughts on forcing an active style on these or bypassing them entirely @hursey013?

Offending elements are:  
```html
<div tabindex="0"
    style="position: absolute; z-index: 0; left: 0px; top: 0px; height: 100%; width: 100%; padding: 0px; border-width: 0px; margin: 0px; cursor: url(&quot;https://maps.gstatic.com/mapfiles/openhand_8_8.cur&quot;), default; touch-action: pan-x pan-y;">
</div>

<iframe aria-hidden="true" frameborder="0"
    style="z-index: -1; position: absolute; width: 100%; height: 100%; top: 0px; left: 0px; border: none;"
    src="about:blank" title="Google Maps"></iframe>
```
